Sources/WalletConnectModal/Modal/Screens/WhatIsWalletView.swift

```diff
@@ -1,29 +1,40 @@
 import SwiftUI
 
 struct WhatIsWalletView: View {
-    
     var navigateTo: (Destination) -> Void
     var navigateToExternalLink: (URL) -> Void
-    
+
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
     var body: some View {
-        
-        VStack(spacing: 10) {
-            HelpSection(
-                title: "A home for your digital assets",
-                description: "A wallet lets you store, send and receive digital assets like cryptocurrencies and NFTs.",
-                assets: [.DeFi, .NFT, .ETH]
-            )
-            HelpSection(
-                title: "One login for all of web3",
-                description: "Log in to any app by connecting your wallet. Say goodbye to countless passwords!",
-                assets: [.Login, .Profile, .Lock]
-            )
-            HelpSection(
-                title: "Your gateway to a new web",
-                description: "With your wallet, you can explore and interact with DeFi, NFTs, DAOs, and much more.",
-                assets: [.Browser, .Noun, .DAO]
-            )
-            
+        content()
+            .onAppear {
+                UIPageControl.appearance().currentPageIndicatorTintColor = AssetColor.foreground1.uiColor
```

Comment:
AppearanceConfigurator could use for setting up appearance()

Reply:
We have AppearanceConfigurator for the sample apps, but not in the modal. This is the only place where we deal with appearance so far. Anyway, I learned that we will end up removing this screen from WalletConnect Modal, and it will be only in Web3Modal with a slightly different design, so I'm tempted to keep it in and deal with it properly when we do Web3Modal design